templates/etc/apache2/sites-available/default.conf.j2

```diff
@@ -12,7 +12,8 @@
 
 {% if item.redirect_http|d() %}
     {{ apache__tpl_macros.get_redirect( item.redirect_http_code|d(307), "/", item.redirect_http) | indent(4) }}
-{% elif item.redirect_to_https|d(True) | bool %}
+{% elif item.redirect_to_https|default(apache__redirect_to_https) | bool %}
```

Comment:
I know what you mean and I generally prefer explicit over implicit and I know that it can be difficult to lookup a macro name of one character but I think that one lookup is worth it as it stands against 4645 uses in DebOps currently.

```
~/.local/share/debops/debops-playbooks/roles $ ag '\|\s*d\(' --stats-only
4645 matches
400 files contained matches
2338 files searched
8921162 bytes searched
``` 

Your voice is very appreciated and valued! Of course we all have our opinions how to do things and I hope we can find a solution we can all agree on.